bower.json

```diff
@@ -8,7 +8,8 @@
   "main": "dist/js/foram_3d.js",
   "dependencies": {
     "dat.gui": "*",
-    "three.js": "threejs#*"
+    "three.js": "threejs#*",
+    "trackball_controls": "https://raw.githubusercontent.com/mrdoob/three.js/master/examples/js/controls/TrackballControls.js"
```

Comment:
To nie jest raczej bardzo dobra praktyka żeby się odwoływać wprost do raw githubowego. Z tego co widzę, dosłownie 10 dni temu pojawiło się coś takiego: https://github.com/gtsop/threejs-trackball-controls - można by z tego skorzystać?